.travis.yml

```diff
@@ -134,6 +134,7 @@ before_script:
 
 script:
   - codecept run unit -f
+  - codecept run etplus_wpunit -f
```

Comment:
I would prefer to move this into the `unit` or `integration` suite and Mock the parts from ET+, solving the download of ET+ here I think is out of scope.

Reply:
When we build out tests, anything we test that ET+ can modify, we test from within ET+ tests themselves. This is not something to be done from within ET tests. We should add whatever wpunit tests that we need with ET+ active right into ET+.

Reply:
@sc0ttkclark that makes sense! I'll look into porting these tests to ET+ and opening a PR there.

Reply:
```suggestion
```